File: src/main/java/test/MoneyUtils.java
package test;

import static test.Main.RUB_CURRENCY;
import static test.Main.USD_CURRENCY;

public class MoneyUtils {
    public static String prettyPrint(Money money) {
        if (money.getCurrency().equals(RUB_CURRENCY)) {
            return String.format("%s₽", money.getValue());
        } else if (money.getCurrency().equals(USD_CURRENCY)) {
            return String.format("%s$", money.getValue());
        } else {
            return String.format("%s %s", money.getValue(), money.getCurrency());
        }
    }
}

File: src/main/java/test/MoneyUtils.kt
package test

import java.math.BigDecimal

fun Number.bd(): BigDecimal = BigDecimal(this.toDouble())

fun Money.prettyPrint(): String = when {
    currency == RUB_CURRENCY -> "$value₽"
    currency == USD_CURRENCY -> "$value$"
    else -> "$value $currency"
}
